packages/component/src/decorator/private/Decorator.tsx

```diff
@@ -13,8 +15,14 @@ const middleware: DecoratorMiddleware[] = [
     (next => request => (request.livestreamingState === 'preparing' ? BorderLoader : next(request)))
 ];
 
-function FluentThemeDecorator({ children }: Readonly<{ readonly children?: ReactNode | undefined }>) {
-  return <DecoratorComposer middleware={middleware}>{children}</DecoratorComposer>;
+const styles = createStyles();
```

Comment:
If we always create styles on importing, then should we `makeCreateStyles` -> `createStyles` instead?

Reply:
I think this was discussed in our early design discussions.

I can't fully understand the proposal. The `makeCreateStyles` allows us to share the logic behind `createStyles` function for each of `xxx/createStyles` module.

Reply:
We can talk about it offline and in another PR.